examples/purchaseTesterTypescript/.node-version

```diff
@@ -0,0 +1 @@
+18
```

Comment:
Should we update to 18.2 here, same as in circleci?

Reply:
this is the version that's suggested by react-native, if a mismatch ends up giving us trouble we might want to have CircleCI match RN instead